pipelineutilities/pipelineutilities/save_standard_json_to_dynamo.py

```diff
@@ -43,7 +44,12 @@ def _save_json_to_dynamo(self, standard_json: dict, expire_time: int) -> bool:
         new_dict = {i: standard_json[i] for i in standard_json if i != 'items'}
         if 'expireTime' not in new_dict:
             new_dict['expireTime'] = expire_time
-        self.standard_json_table.put_item(Item=new_dict)
+        try:
+            self.standard_json_table.put_item(Item=new_dict)
```

Comment:
Just a suggestion. I'd remove this from the constructor
```
        self.dynamodb = boto3.resource('dynamodb')
        self.standard_json_table = self.dynamodb.Table(self.config['standard-json-tablename'])
```
add this to the imports:
```
from botocore.exceptions import ClientError
```
and put that update this method _save_json_to_dynamo
```
        try:
            standard_json_table = boto3.resource('dynamodb').Table(self.config['standard-json-tablename'])
            standard_json_table.put_item(Item=new_dict)
        except ClientError as ce:
            success_flag = False
            capture_exception(ce)
            print(f"Error saving to {self.config['standard-json-tablename']} table - {ce.response['Error']['Code']} - {ce.response['Error']['Message']}")
```

I like this flow because all db operations and error handling are encapsulated here. You're currently not doing any error handling in the constructor which could fail if the database does not exist. However there _could_ be a performance hit(idk) since it'd be calling "boto3.resource('dynamodb').Table" every time this method would be called. You'd have to test it under load since I have no idea.